src/Selenium2Library/__init__.py

```diff
@@ -34,13 +34,22 @@ class Selenium2Library(
     imported into your Robot test suite (see `importing` section), and the 
     `Open Browser` keyword must be used to open a browser to the desired location.
 
-    = Locating elements =
+    **--- Note important change starting with Version 1.7.0 release ---**
+    *Locating or getting elements*
+    *Locating or specifying elements*
```

Comment:
Are these two lines accidental duplication, or are they on purpose? It seems like you either want 38 or 39 but not both. Also, why did you remove the = sign in favor of asterisks?